internal/controller/isbservicerollout_controller.go

```diff
@@ -201,13 +205,12 @@ func (r *ISBServiceRolloutReconciler) reconcile(ctx context.Context, isbServiceR
 
 	existingISBServiceDef, err := kubernetes.GetCR(ctx, r.restConfig, newISBServiceDef, "interstepbufferservices")
 	if err != nil {
-		// create object as it doesn't exist
+		// create an object as it doesn't exist
 		if apierrors.IsNotFound(err) {
 			numaLogger.Debugf("ISBService %s/%s doesn't exist so creating", isbServiceRollout.Namespace, isbServiceRollout.Name)
 			isbServiceRollout.Status.MarkPending()
 
-			err = kubernetes.CreateCR(ctx, r.restConfig, newISBServiceDef, "interstepbufferservices")
-			if err != nil {
+			if err := kubernetes.CreateCR(ctx, r.restConfig, newISBServiceDef, "interstepbufferservices"); err != nil {
```

Comment:
should we be calling `r.ErrorHandler()` here?

Reply:
No, as it's already called via parent